livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -105,8 +90,21 @@ class RealtimeToolCall:
     """id of the tool call"""
 
 
-# TODO(theomonnom): add the content type directly inside RealtimeContent?
-# text/audio/transcript?
+@dataclass
+class Capabilities:
+    supports_chat_ctx_manipulation: bool
```

Comment:
I thinks since this is temporary, we shouldn't care about implementing Capabilities today? wdyt?

Reply:
How do we handle truncation in baseclass then?

Reply:
both OpenAI and Google doesn't use it right now?
`supports_chat_ctx_manipulation=False`

The motivation here is that I want to avoid adding new APIs that may be hard to remove in the future as it'll be a breaking change